server/lib/matchmaking/matchmaking-service.ts

```diff
@@ -344,7 +345,7 @@ export class MatchmakingService {
       matchInfo,
       clients,
       slots,
-      setup = {},
+      setup = { gameId: '', seed: 0 },
```

Comment:
So *this* was the part of the `Partial<>` change I had a problem with, and you didn't roll this back. Either roll this back as well, or do the necessary investigation to figure out why any of this is optional in the first place

Reply:
I'm just gonna revert it for now and investigate at some later point. Gonna file an issue for it.